spec.md

```diff
@@ -2,15 +2,15 @@
 # SPDX-FileCopyrightText: 2019 Free Software Foundation Europe e.V.
 # SPDX-License-Identifier: CC-BY-SA-4.0
 
-title: "REUSE Specification – Version 3.1"
+title: "REUSE Specification – Version 3.2"
 ---
 
-This specification defines a standardized method for declaring copyright and
+This specification defines a standardised method for declaring copyright and
 licensing for software projects. The goal of the specification is to have
-unambiguous, human- and machine-readable copyright and licensing information for
-each individual file in a project. Ideally this information is embedded into
-every file, so that the information is preserved when the file is copied and
-reused by third parties.
+comprehensive, unambiguous, human- and machine-readable copyright and licensing
+information for each individual file in a project. Ideally this information is
```

Comment:
Adding a comment at the first instance of 'copyright and licensing information' in this file.

I rather hate that mouthful, and it's awkward to write in text, especially when the word 'and' makes the sentence ambiguous.

I was recently made aware of the term 'Copyright Management Information (CMI)' used in US law, notably the DMCA. According to [Copyright Alliance](https://copyrightalliance.org/education/copyright-law-explained/the-digital-millennium-copyright-act-dmca/copyright-management-information/), CMI 'would include the copyrighted work’s title, ISBN number or copyright registration number; **the copyright owner’s name**; the creator’s name. name; **and terms and conditions** for use of the work.'

I don't know if it makes sense to change the term 'Copyright and Licensing Information' with 'Copyright Management Information (CMI)' because we only intend to mean a subset of CMI, but I would love love love love to change the wording.

Reply:
I get that and I agree that a single expression word would help phrasing documentation. Some suggestions that came to my mind:

- **Annotations**: more focused on how it is used, but can be a word that is used consistently in the REUSE documentation.
- **License information**: when speaking I often focus on the license information, taking the necessary copyright information as implied/included. Given that copyright and license headers are separate I can see how this can lead to confusion.
- **Legal information**: a broader term that might include more information and is less precise than the explicit mention of copyright and license information.

Reply:
I am lukewarm to 'annotations', but do not like that it is plural, and do not like that it doesn't contain the word 'licensing' or 'copyright' or some such.

I do not like 'license information' because sometimes, occasionally, you want to talk about license information to the _exclusion_ of copyright information.

'Legal information' seems too broad to me indeed.

I am raising this here, but I would like to use the term we decide on _outside_ of the spec as well.

Internally in the tool, I call this information '**REUSE Information**', but 'REUSE Information' has a little more metadata than just 'copyright and licensing' that may or may not be relevant, specifically _other SPDX tags_ (currently only `SPDX-FileContributorText`).

Reply:
What about just “licensing information”?

That should imply copyright info a bit stronger, since “license information” could be read as limited to just the license.